apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -593,11 +639,15 @@ def create_table_for_html_display(m, n, name, label, data, format_type):
             )
             display_array += '<td class="allborder">'
             display_array += escape(raw_submitted_answer)
-            if feedback_each_entry is not None:
+            if feedback_each_entry is not None and isinstance(
+                feedback_each_entry, dict
+            ):
                 if feedback_each_entry[each_entry_name] == "correct":
                     feedback_message = '&nbsp;<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i></span>'
                 elif feedback_each_entry[each_entry_name] == "incorrect":
                     feedback_message = '&nbsp;<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i></span>'
+                else:
+                    raise ValueError("invalid feedback type: this should not happen")
```

Comment:
Nit, include the feedback type value in the error message to aid in debugging if it _does_ happen?